tools/locate_bins.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
```

Comment:
its prefered to add:
```
#!/bin/env bash
```
Also always in new scripts add:
```
set -euo pipefail
```
Not only
```
set -e
```

Reply:
`#!/bin/bash` seems to be the norm in scripts (configure.sh included) so I'm going to go with that.  `set -euo pipefail` is added.

Reply:
As for why use bin env vs no bin env - here you have the explanation. https://stackoverflow.com/a/16365367/4239386
Thats a minor.

`-e` is important - you need commands to fail - then **catch the fail** explicitly.

Reply:
Well the problem is that `/bin/env` doesn't exist but `/usr/bin/env` does.  For compatibility I'm sticking with `/bin/bash`

I've looked more into how `-e` works and now made sure to include it's use.